snapshot/lib/snapshot/runner.rb

```diff
@@ -70,6 +81,56 @@ def work
       end
     end
 
+    def patch_scheme_to_save_screenshots
+      scheme = Snapshot.config[:scheme]
+      scheme_absolute_path = File.join(File.expand_path(".", Snapshot.project.path), 'xcshareddata', 'xcschemes', scheme + ".xcscheme")
+
+      scheme_xml_doc = fetch_scheme_xml_file(scheme_file_path: scheme_absolute_path)
+
+      backup_scheme_file_path = scheme_absolute_path + "_original"
+      backup_scheme(scheme_absolute_path: scheme_absolute_path, backup_scheme_absolute_path: backup_scheme_file_path)
+
+      scheme_elements = scheme_xml_doc.elements['Scheme']
+      test_action = scheme_elements.elements['TestAction']
+      test_action.attributes["systemAttachmentLifetime"] = "keepAlways"
```

Comment:
Can we modify this behavior without requiring a certain structure of the file? Could we somehow use a search and replace instead, so that if  changes the format, our tools don't break?